<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>921</width>
     <height>721</height>
    </rect>
   </property>
   <widget class="QFrame" name="LoginBox">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>90</y>
      <width>311</width>
      <height>411</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border:2px solid rgba(0,0,0,0);
border-bottom-color:rgba(46, 82, 101, 200);
color:rgb(0,0,0);
padding-bottom:7px;
border-radius: 15px;

</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>4</number>
    </property>
    <widget class="QPushButton" name="loginBtn">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>250</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#loginBtn{
	background-color:rgba(85, 98, 112, 255);
	color:rgba(255,255,255,200);
	border-radius:5px;
}

QPushButton#loginBtn:pressed{
	padding-left:5px;
	padding-top:5px;
	background-color:rgba(255,107,107,255);
	background-position:calc(100% - 10px)center;
}

QPushButton#loginBtn:hover{
	background-color:rgba(255,107,107,255);
}</string>
     </property>
     <property name="text">
      <string>Sign In</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="password">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>180</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="username">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>130</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255 ,255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Sign In</string>
     </property>
    </widget>
    <widget class="QPushButton" name="closeBtn">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:  rgba(85, 98, 112, 255);
background-color: #FBAB7E;

</string>
     </property>
     <property name="text">
      <string>x</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="error">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>310</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>381</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: #FBAB7E;
background-image: linear-gradient(62deg, #FBAB7E 0%, #F7CE68 100%);


border-radius: 15px;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
